corehq/apps/reports/standard/cases/basic.py

```diff
@@ -65,25 +65,30 @@ def _build_query(self):
         if self.case_status:
             query = query.is_closed(self.case_status == 'closed')
 
-        if self.request.can_access_all_locations and (
-                EMWF.show_all_data(mobile_user_and_group_slugs)
-                or EMWF.no_filters_selected(mobile_user_and_group_slugs)
-        ):
-            pass
+        case_onwer_filters = []
```

Comment:
I think this is a spelling error, right? 
```suggestion
        case_owner_filters = []
```

Reply:
Yes. It is a typo. Thanks for catching it 😄

Reply:
Fixed.